easybuild/easyconfigs/g/Gurobi/Gurobi-9.0.1-GCCcore-9.3.0-Python-3.8.2.eb

```diff
@@ -0,0 +1,38 @@
+name = 'Gurobi'
+version = '9.0.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://www.gurobi.com'
+description = """The Gurobi Optimizer is a state-of-the-art solver for mathematical programming.
+The solvers in the Gurobi Optimizer were designed from the ground up to exploit modern
+architectures and multi-core processors, using the most advanced implementations of the
+latest algorithms."""
+
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
+
+source_urls = ['https://packages.gurobi.com/%(version_major_minor)s/']
+sources = ['%(namelower)s%(version)s_linux64.tar.gz']
+patches = ['%(name)s-%(version)s_use-eb-python-gurobi-shell.patch']
+checksums = [
+    '17e2facda111180eee61eeded0b8716230bbe09faa7c61356dc79f002ff86cb7',  # gurobi9.0.1_linux64.tar.gz
+    'b4a998182d05f969d1de519f4746ac9e0c6646dd35233231b6ab5963dfa67d01',  # Gurobi-9.0.1_use-eb-python-gurobi-shell.patch
+]
+
+builddependencies = [
+    ('binutils', '2.34'),
+]
+
+dependencies = [
+    ('Python', '3.8.2'),
+]
+
+# remove bundled Python interpreter in favour of the dependency in EB
+postinstallcmds = ['rm %(installdir)s/bin/python3.7']
```

Comment:
@lexming Maybe this should be a bit more generic?

```python
postinstallcmds = ['rm %(installdir)s/bin/python*']
```

Reply:
fixed :+1: